EnglishFilter/sections/general_url.txt

```diff
@@ -53,6 +53,17 @@
 ! Sample URL: ||js.passaro-de-fogo.biz/t/113/750/a1113750.js
 /\/t\/[0-9]{3}\/[0-9]{3}\/a[0-9]{4,9}\.js$/$script
 !
+! generic regex
```

Comment:
It is for Popads, right?

Reply:
TBH I don't know. To me PopAds is those caught by `("abort-on-property-write", "_pop")` and their URL pattern is different.

Reply:
:) Ok, please show few urls, matched by these regexes.

Reply:
```
https://regex101.com/r/tl281M/1
https://regex101.com/r/bZA6Vu/1
```

Reply:
Can't open that scripts:)
Do you have their saved content? Just want to look.

Reply:
Saving the page as html doesn't work, maybe try another browser? @zloyden @AdamWr @Sergey-Lyapin Can any of you open the links?

Reply:
I mean I can't open ad scripts from tests) regex101.com works fine.

Reply:
What do you mean by saying can't open? Do you see my Unit tests? if you see, then you can open those script URLs by copy & paste e.g. `https://alterupholdcinema.com/51/5f/e0/515fe0778757eae5228770cf1427e721.js`

Reply:
I suggest to place new rules under directive, without removing current rules
```
!#if (adguard_app_ios || adguard_ext_safari)

!#endif
```

Reply:
Do you mean put them back to specific.txt?

Reply:
@Yuki2718 hi! I've looked for your conservation and I guess what you should do:
1. Revert the old rules
2. Move new rules that have `!+ PLATFORM(ios, ext_safari, ext_android_cb)` to the `specific.txt`, but without hints, use the directive.

Reply:
Done.